sources/packages/backend/apps/queue-consumers/src/queues-metrics.module.module.ts

```diff
@@ -0,0 +1,25 @@
+import { Global, LoggerService, Module, OnModuleInit } from "@nestjs/common";
+import { InjectLogger } from "@sims/utilities/logger";
+import { MetricsService } from "./services";
+
+@Global()
+@Module({
+  providers: [MetricsService],
+  exports: [MetricsService],
+})
+export class QueuesMetricsModule implements OnModuleInit {
+  constructor(private readonly metricsService: MetricsService) {}
```

Comment:
Do we require this un-used service at constructor?

Reply:
Got it now. The comment was added to the wrong file. I will remove it from the controller.